data-model/SDP-data-model-templates/textual-artifact_sdp-template.json

```diff
@@ -7,6 +7,20 @@
   ],
   "extent": "",
   "embodies": "URI of Work Authority File",
+  "attested_title": {
+    "transcription": [
+      {
+        "syr": "field name must be an ISO language code"
+      }
+    ],
+    "translation": [
+      {
+        "en": "field name must be an ISO language code"
+      }
+    ],
+    "locus": ""
+  },
+  "supplied_title": "",
```

Comment:
This is like the 'title from RA' field in our Airtable form. I don't know if we will need this or if we will require users to create an authority record? I like the idea of having a less committal option for titling things (this would be, for example, where we'd put 'unidentified')